renovate.json

```diff
@@ -1,37 +1,18 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:base", "schedule:daily", ":disableRateLimiting"],
+  "extends": [
+    "config:base",
+    "schedule:daily",
+    ":disableRateLimiting"
+  ],
   "schedule": "before 5am every weekday",
   "timezone": "Europe/Berlin",
-  "packageRules": [
-    {
-      "matchUpdateTypes": ["minor", "patch"],
-      "matchCurrentVersion": ">= 1.0.0",
-      "automerge": true
-    },
-    {
-      "matchDepNames": ["node", "pnpm"],
-      "enabled": false
-    },
-    {
-      "matchPackagePrefixes": ["@types/"],
-      "automerge": true,
-      "major": {
-        "automerge": false
-      }
-    }
-  ],
-  "lockFileMaintenance": {
-    "enabled": true,
-    "automerge": false
-  },
-  "major": {
-    "automerge": false
+  "rebaseWhen": "never",
+  "vulnerabilityAlerts": {
```

Comment:
```suggestion
  "vulnerabilityAlerts": {
  "enabled": true,
```

adding enables makes sure that dependency updates that we disabled aren't blocked if they have security updates.

Reply:
good point. don't know why I missed this egain... ;-) thx.